AppLovin MAX Demo App - Swift/AppLovin MAX Demo App - Swift/MAX/App Open/ALMAXAppOpenAdViewController.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  ALMAXAppOpenAdViewController.swift
+//  AppLovin MAX Demo App - Swift
+//
+//  Created by Avi Leung on 2/13/23.
+//  Copyright © 2023 AppLovin. All rights reserved.
+//
+
+import UIKit
+import Adjust
+import AppLovinSDK
+
+class ALMAXAppOpenAdViewController: ALBaseAdViewController, MAAdViewAdDelegate, MAAdRevenueDelegate
+{
+    private let appOpenAd = MAAppOpenAd(adUnitIdentifier: "YOUR_AD_UNIT_ID")
+    
+    // MARK: View Lifecycle
+    
+    override func viewDidLoad()
+    {
+        super.viewDidLoad()
+        
+        appOpenAd.delegate = self
+        appOpenAd.revenueDelegate = self
+        
+        // Load the first ad
+        appOpenAd.load()
+    }
+    
+    // MARK: IB Actions
+    
+    @IBAction func showAd()
+    {
+        appOpenAd.show()
+    }
+    
+    // MARK: MAAdDelegate Protocol
+    
+    func didLoad(_ ad: MAAd)
+    {
+        // App Open ad is ready to be shown. '[self.appOpen isReady]' will now return 'YES'
```

Comment:
nit: Convert Objective-C code in comment to Swift (looks like the code you copied from had it too)
```suggestion
        // App Open ad is ready to be shown. 'appOpenAd.isReady' will now return 'true'
```